[NEWTOOL] Labs Sketchfab USDZ Apply Animation Alias

WaffleBoyTom opened the pull request:
Labs Sketchfab USDZ Apply Animation is an alias that spawns a python script lop. It applies your animation to your USDSkel as this is an issue with the USDZ files coming from Sketchfab.

WaffleBoyTom committed:
Cool stuff

WaffleBoyTom committed:
Delete whatamidoing.png

WaffleBoyTom committed:
Merge branch 'sideeffects:Development' into Development

WaffleBoyTom committed:
Merge branch 'sideeffects:Development' into Development

WaffleBoyTom committed:
[NEWTOOL] Added *Sketchfab USDZ Apply Animation Alias*.

 This alias creates a python script lop that fixes an issue with USDZ animation from Sketchfab.

WaffleBoyTom committed:
Delete aliases.shelf

WaffleBoyTom committed:
[NEWTOOL] - Sketchfab USDZ Apply Animation alias

 This alias creates a pythonscript lop that fixes an issue with USDZ animation coming from Sketchfab

WaffleBoyTom committed:
[NEWTOOL] LOPs Alias called Sketchfab USDZ Apply Animation that fixes an issue with Sketchfab's USDZ assets where the animation is not linked to the skeleton

WaffleBoyTom committed:
[NEWTOOL] Labs Sketchfab USDZ Apply Animation applies animation to animated USDZ files coming from Sketchfab

aomai01 reviewed (changes requested):
Some changes are required. I will communicate this in a separate channel.

WaffleBoyTom committed:
[NEWTOOL] Adds Labs Sketchfab USDZ Apply Animation which applies animation to USDZ from Sketchfab by binding the Skeleton prim to the SkelAnimation prim

WaffleBoyTom committed:
[NEWTOOL] Adds Labs Mandelbulb Generator which creates a mandelbulb geometry or volume

WaffleBoyTom committed:
[NEWTOOL] **Labs Sketchfab USDZ Apply Animation**

Alias that creates a python script LOP that applies animation to a USDZ file coming from Sketchfab
